pyiode/pyiode_ws.pyx

```diff
@@ -132,32 +133,80 @@ def ws_content(pattern: str = '*', objtype: int = 6) -> List[str]:
             res[nb] = pystr(s)
             nb = nb + 1
 
-    free_tbl(cnt)
+    SCR_free_tbl(cnt)
 
     return res
 
-def ws_content_cmt(pattern: str = '*') -> List[str]:
+def ws_content(pattern: Union[str, List[str]] = '*', objtype: int = 6) -> List[str]:
+    r"""Return the names of objects of a given type, satisfying pattern specification(s). 
+    The resulting list is sorted in alphabetic order.
+
+    Parameters
+    ----------
+    pattern: string or list of strings
+        each string may contain wildcards characters like '*' or '?'.
+        Default '*', meaning "all objects".
+
+    objtype: int
+        IODE object type (0-6, 0 for comments...)
```

Comment:
(0-6, 0 for comments...) --> (K_CMT, K_EQS, K_IDT, K_LST, K_SCL, K_TBL, K_VAR for Comments ... Variables)